<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_31">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_41">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_51">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_61">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_71">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_81">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_91">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_enter">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_solve">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>solve</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_32">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_42">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_72">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_92">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>490</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_52">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_62">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_82">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_22">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_12">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_33">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_43">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_73">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_93">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>490</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_53">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_63">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_83">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_23">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_13">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_34">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_44">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_74">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_94">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>490</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_54">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_64">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_84">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_24">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_14">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_35">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_45">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_75">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_95">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>490</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_55">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_65">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_85">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_25">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_15">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_36">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_46">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_76">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_96">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_56">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_66">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_86">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>430</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_26">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_16">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_37">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_47">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_77">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_97">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>490</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_57">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_67">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_87">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>430</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_27">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_17">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_38">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_48">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_78">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_98">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>490</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_58">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_68">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_88">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>430</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_28">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_18">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_39">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_49">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_79">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_99">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>490</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_59">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_69">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_89">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>430</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_29">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_19">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>356</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>10</y>
      <width>21</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>51</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>Datei</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBearbeiten">
    <property name="title">
     <string>Bearbeiten</string>
    </property>
    <addaction name="actionclear_all"/>
   </widget>
   <addaction name="menumenu"/>
   <addaction name="menuBearbeiten"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionclear_all">
   <property name="text">
    <string>clear all</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
